pom.xml

```diff
@@ -73,6 +73,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.mule.runtime</groupId>
+            <artifactId>mule-module-logging</artifactId>
```

Comment:
What's the purpose of this one?

Reply:
We init a couple of loggers but we never use them,, I will remove it to keep the code clean,, we can add it again if needed